CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -1014,6 +1014,29 @@ public CommandBase OpenRDP
         }
 
         /// <summary>
+        /// Get SSH settngs for the selected ComputeNode
+        /// </summary>
+        public CommandBase GetSSH
```

Comment:
This was following our (inconsistent) capitalisation of RDP.  If memory serves, MainViewModel uses the capitalisation "RDP", and everywhere else uses the BCL-style "Rdp", and I followed that in each file.